client/landing/gutenboarding/index.tsx

```diff
@@ -172,3 +176,29 @@ function ensureRandomizedDesignsAreUpToDate() {
 function isDeepEqual( stored: Design[], available: Design[] ): boolean {
 	return isEmpty( xorWith( stored, available, isEqual ) );
 }
+
+/**
+ * Pre-fill Anchor.fm podcast title if available
+ */
+async function setPodcastTitle() {
+	// Feature flag 'anchor-fm-dev' is required for anchor podcast id to be read
+	if ( ! config.isEnabled( 'anchor-fm-dev' ) ) {
+		return;
+	}
+
+	const anchorFmPodcastId = new URLSearchParams( window.location.search )
+		.get( 'anchor_podcast' )
+		?.replace( /[^a-zA-Z0-9]/g, '' );
+
+	if ( ! anchorFmPodcastId ) {
+		return;
+	}
+
+	await apiFetch( {
```

Comment:
Could the `wpcomRequest()` method can be used in replacement of `apiFetch()`? That way the hostname `public-api.wordpress.com` can be omitted.

Reply:
@yansern I'm not finding that `wpcomRequest()` is currently used outside of the `@automattic/data-stores` package, or part of the package API. Let me know if I missed something! That would be a better option if available.